cmd/osctl/pkg/client/client.go

```diff
@@ -271,3 +271,22 @@ func (c *Client) Top() (pl []proc.ProcessList, err error) {
 	err = dec.Decode(&pl)
 	return
 }
+
+// DF implements the proto.OSDClient interface.
+func (c *Client) DF() (err error) {
+	ctx := context.Background()
+	reply, err := c.client.DF(ctx, &empty.Empty{})
+	if err != nil {
+		return
+	}
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+	fmt.Fprintln(w, "FILESYSTEM\tSIZE(GB)\tUSED(GB)\tAVAILABLE(GB)\tPERCENT USED\tMOUNTED ON")
```

Comment:
hard to say without playing with it, but does it allow for better alignment if you specify width, like `%6.02f`, (for perecent that should be enough for example)

Reply:
The alignment _seems_ ok? I can play with it.

Reply:
I think the values in your example above look to be left-aligned, not right-aligned like `df` does, so that dots are under dots, and same-order digits are under each other. nitpicking on my side ;)

Reply:
Ahh I see. Hmm, I'm only hesitant to right align because in the rest of `osctl` we don't. In its' current form it is consistent with the rest of the UI. Maybe we can have a follow-up PR for getting the `osctl` outputs aligned the way we want?

Reply:
ah, got it, then it's good :+1: